ingest.go

```diff
@@ -858,7 +858,7 @@ func ingestTargetLevel(
 //     local sstables. This is the step where overlap with memtables is
 //     determined. If there is overlap, we remember the most recent memtable
 //     that overlaps.
-//  6. Update the sequence number in the ingested local sstables. (Shared
```

Comment:
I think this will stay Shared, because external sstables are assigned sequence numbers the same way as local sstables.